pkg/collect/logs.go

```diff
@@ -117,7 +117,19 @@ func getPodLogs(ctx context.Context, client *kubernetes.Clientset, pod corev1.Po
 		podLogOpts.TailLines = &limits.MaxLines
 	}
 
-	if limits != nil && limits.MaxAge != "" {
+	if limits != nil && limits.SinceTime != "" {
+		t, err := time.Parse(time.RFC3339, limits.SinceTime)
+		if err != nil {
+			logger.Printf("unable to parse --since-time=%s\n", limits.SinceTime)
+		}
+
+		sinceTime := metav1.NewTime(t)
```

Comment:
this should be in an `else` statement to handle the case when parsing `SinceTime` fails.

Reply:
Done!